sns_ik_lib/include/sns_ik/sns_ik.hpp

```diff
@@ -64,16 +68,14 @@ namespace sns_ik {
     int CartToJnt(const KDL::JntArray &q_init, const KDL::Frame &p_in, KDL::JntArray &q_out, const KDL::Twist& bounds=KDL::Twist::Zero());
     int CartToJnt(const KDL::JntArray& q_in, const KDL::Twist& v_in, KDL::JntArray& qdot_out);
 
-    inline void SetSolveType(SolveType type) {
-      m_solvetype = type;
-    }
+    bool setVelocitySolveType(VelocitySolveType type, double loop_rate);
```

Comment:
Why is loop_rate involved with setVelocitySolveType instead of being a separate setter?